wbia/other/ibsfuncs.py

```diff
@@ -1244,7 +1244,7 @@ def check_cache_purge(ibs, ttl_days=90, dryrun=True, squeeze=True):
         python -m wbia.other.ibsfuncs --exec-check_cache_purge
 
     Example:
-        >>> # ENABLE_DOCTEST
+        >>> # DISABLE_DOCTEST
```

Comment:
I think we're seeing a legitimate error here...  Shouldn't we fix the code instead of disabling the test?

```
DOCTEST TRACEBACK
Traceback (most recent call last):
  File "/virtualenv/env3/lib/python3.7/site-packages/xdoctest/doctest_example.py", line 598, in run
    exec(code, test_globals)
  File "<doctest:/wbia/wildbook-ia/wbia/other/ibsfuncs.py::check_cache_purge:0>", line rel: 5, abs: 1251, in <module>
    >>> result = check_cache_purge(ibs)                                                                File "/wbia/wildbook-ia/wbia/other/ibsfuncs.py", line 1477, in check_cache_purge
    db.squeeze()
  File "/wbia/wildbook-ia/wbia/dtool/sql_control.py", line 726, in squeeze
    self.shrink_memory()
  File "/wbia/wildbook-ia/wbia/dtool/sql_control.py", line 708, in shrink_memory
    self.connection.commit()
AttributeError: 'Connection' object has no attribute 'commit'
DOCTEST REPRODUCTION
CommandLine:
    pytest /wbia/wildbook-ia/wbia/other/ibsfuncs.py::check_cache_purge:0
```

Reply:
#141 should fix the test

Reply:
This doctest needs to be disabled for now.  Finding the error was coincidental and opportunistic, but also tangential to running a test on this function.  This function is a database maintenance script that does is functionally a no-op (or worse destructive) on test databases.

Reply:
I misread the test error for this PR, thinking it was erroring for a different, benign reason.  I'm glad this error was caught on a more thorough review, but the doctest still needs to be disabled going forward.  If you want, we can add a new doctest for this error directly in dtool to complete the circle on the fix.

Reply:
Yeah it would be better if it's replaced with a different test....